x-pack/packages/security-solution/navigation/src/landing_links/translations.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+
+export const LANDING_LINKS_ACCORDION_HEADER = i18n.translate(
+  'xpack.securitySolution.navigation.landingLinks',
```

Comment:
https://buildkite.com/elastic/kibana-pull-request/builds/216762#0190311f-ce6f-4723-80e4-40ddd1ddbf67


It complains about the naming here:

  | Error: Expected "xpack.securitySolution.navigation.landingLinks" id to have "securitySolutionPackages" namespace. 



Please have a look to the example keys here: x-pack/packages/security-solution/data_table/components/data_table/column_headers/helpers.tsx line 157